src/main/scala/com/gu/zuora/creditor/CreditTransferService.scala

```diff
@@ -47,16 +47,23 @@ class CreditTransferService(
   import CreditTransferService._
   import ModelReaders._
 
-  def processExportFile(exportId: ExportId): Int = {
+  def processExportFile(exportId: ExportId): AdjustmentsReport = {
     val maybeExportCSV = downloadGeneratedExportFile(exportId)
+    maybeExportCSV.foreach{ rawCSV =>
+      logger.info("==========================================")
+      logger.info(s"csv export: $rawCSV")
```

Comment:
probs could log this on its own logger line

Reply:
yeah that was my debugging in progress ...